### DISCUSSION TITLE
Open At Login on Mac

### DISCUSSION THREAD

**Discussion by 44jessejames44** [iterate-ch/cyberduck#15784] [Q&A]
How can I stop Cyberduck from opening at login on an Intel iMac?  Running OS Monterey 12.7.4  Even with the option unchecked, it still opens every time I log in.  I've checked and then unchecked the option, and nothing will make it stop

This ticket system is to report bugs and feature requests. For support, visit the [help page](https://cyberduck.io/help/) first.

 * If you have trouble connecting to a server or your login credentials are not valid, try to resolve the issue with the assistance of your hosting service provider. Any such ticket will be closed with a resolution of _thirdparty_.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. macOS]

**Log Files**
Please include the [log file](https://docs.cyberduck.io/cyberduck/support/#logging-output). You can reach the logging output in _Preferences → Connection_.

**Additional context**

**Comment by dkocher** [comment]
I assume you are referring to _Mountain Duck_. You can remove _Mountain Duck_ in _System Settings → Login Items_.
